playground/backend/containers/go/Dockerfile

```diff
@@ -76,9 +76,11 @@ RUN sh /proxy.sh
 # Create a user group `appgroup` and a user `appuser`
 RUN groupadd --gid 20000 appgroup \
   && useradd --uid 20000 --gid appgroup --shell /bin/bash --create-home appuser
-# Chown all the files to the app user.
-RUN chown -R appuser:appgroup /opt/playground/backend/ && chown -R appuser:appgroup /opt/playground/prepared_folder/ \
-  && chmod +x /entrypoint.sh
+
+RUN mkdir -p /opt/playground/backend/executable_files/
```

Comment:
Can we change the path to `PREPARED_MOD_DIR` env

Reply:
And in other cases too.

Reply:
`PREPARED_MOD_DIR` equals `/opt/playground/prepared_folder/`, but I need `/opt/playground/backend/executable_files/`

Reply:
sry, i meant executable_files to `PIPELINES_FOLDER_NAME`

Reply:
`PIPELINES_FOLDER_NAME` equals `executable_files`, but I need `/opt/playground/backend/executable_files/`